procedures/foreman/release.md.erb

```diff
@@ -4,7 +4,7 @@
 
 * Release Owner: <%= owner %>
 * Release Engineer: <%= engineer %>
-<% unless is_rc && full_version.end_with?('1') -%>
```

Comment:
Technically this is a change, since it previously matched on `rc11` too, but that was a bug which we never hit since we've never done so many RCs.